core/dccontracts/upgrader.go

```diff
@@ -70,7 +69,7 @@ func init() {
50565b6000602082019050611d9d6000830184611beb565b92915050565b6000611dad611dbe565b9050611db9828261203f565b919050565b6000604051905090565b600067ffffffffffffffff821115611de357611de26121d2565b5b611dec82612215565b9050602081019050919050565b6000819050602082019050919050565b60008190508160005260206000209050919050565b600081519050919050565b600081519050919050565b6000602082019050919050565b600082825260208201905092915050565b600082825260208201905092915050565b600081905092915050565b6000611e7982611fc1565b9150611e8483611fc1565b9250827fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff03821115611eb957611eb86120e7565b5b828201905092915050565b6000611ecf82611fc1565b9150611eda83611fc1565b925082611eea57611ee9612116565b5b828204905092915050565b6000611f0082611fc1565b9150611f0b83611fc1565b9250817fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff0483118215151615611f4457611f436120e7565b5b828202905092915050565b6000611f5a82611fc1565b9150611f6583611fc1565b925082821015611f7857611f776120e7565b5b828203905092915050565b6000611f8e82611fa1565b9050919050565b60008115159050919050565b600073ffffffffffffffffffffffffffffffffffffffff82169050919050565b6000819050919050565b82818337600083830152505050565b60005b83811015611ff8578082015181840152602081019050611fdd565b83811115612007576000848401525b50505050565b6000600282049050600182168061202557607f821691505b6020821081141561203957612038612145565b5b50919050565b61204882612215565b810181811067ffffffffffffffff82111715612067576120666121d2565b5b80604052505050565b600061207b82611fc1565b91507fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff8214156120ae576120ad6120e7565b5b600182019050919050565b60006120c4826120cb565b9050919050565b60006120d682612226565b9050919050565b6000819050919050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052601160045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052601260045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052602260045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052603160045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052603260045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052604160045260246000fd5b600080fd5b600080fd5b600080fd5b600080fd5b6000601f19601f8301169050919050565b60008160601b9050919050565b7f4f776e61626c653a206e6577206f776e657220697320746865207a65726f206160008201527f6464726573730000000000000000000000000000000000000000000000000000602082015250565b7f4f6e6c79206f776e65722063616e2063616c6c2066756e6374696f6e00000000600082015250565b7f466f726269640000000000000000000000000000000000000000000000000000600082015250565b7f4f6e6c79207369676e65722063616e2063616c6c2066756e6374696f6e000000600082015250565b61230681611f83565b811461231157600080fd5b50565b61231d81611fc1565b811461232857600080fd5b5056fea26469706673582212208879968f26c5ad21d7e3b58081ca73de324deb7256e1760798788f290bffec8364736f6c63430008060033",
```

Comment:
Why do you delete our upgraded codes?
The byte codes are totally different after some hard forks.
I think you miss out the most important part.

Reply:
My mistake, I misunderstood the process of upgrading the contract and initialization, now use `go:generate` to convert the contract hex string to a binary file, and generate `go:embed` code to embed binary file in the execution file